documentation/synthetic_tokens/tutorials/liquidation.md

```diff
@@ -120,6 +134,8 @@ PRICE_FEED_CONFIG={"type":"medianizer","apiKey":"YOUR_API_KEY","pair":"ethbtc","
 
 The parameters above, as well as other optional parameters are explained in the appendix of this tutorial. **Be sure to add in your mnemonic and your crypto watch API key.** The parameter in the example above conform to [UMIP-2](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-2.md#implementation)'s specification.
 
+Note that the `EMP_ADDRESS` above currently refers to the ETHBTC synthetic token deployed on Kovan. The EMP refers to the `ExpiringMultiParty` financial contract used here.
```

Comment:
this is great! would also be nice to add a link to the deployed addresses section in the docs perhaps?

following on from this it also looks like these other two docs sites are out of date:
https://docs.umaproject.org/uma/oracle/mainnet_deployment_info.html
https://docs.umaproject.org/uma/developer_reference/contract_addresses.html

I've created an issue for these two points specifically to keep track of. This can be found here: #1652

Reply:
I think that's a good idea, but we should probably revisit it when we move to a new docs site.